providers/hp/ilo/helpers.go

```diff
@@ -135,6 +137,18 @@ func (i *Ilo) cmpAccessSettings(cfg *cfgresources.Network) (AccessSettings, bool
 		if currentConfig.VirtualMediaPort != cfg.KVMMediaPort {
 			return false
 		}
+		// Comparing SNMP settings for iLO 4 and iLO5
+		switch i.HardwareType() {
```

Comment:
I have applied the constants in both cases.

Sorry, I don't understand what you mean with "move this the blocks under the case conditions into its own function". 

This section compares SNMP settings between the declared config and the config present on the BMC. Would you like me to make something like

`compareSNMPIlo4(...)` and `compareSNMPIlo5(...)` to avoid those long lines?

Reply:
> I have applied the constants in both cases.
here I mean it could be `case Ilo4:` instead of `case "Ilo4"`.

> `compareSNMPIlo4(...)` and `compareSNMPIlo5(...)` to avoid those long lines?
yep 👍